app/localShortcuts.js

```diff
@@ -20,7 +20,9 @@ module.exports.register = (win) => {
     ['CmdOrCtrl+Alt+Left', messages.SHORTCUT_PREV_TAB],
     ['Ctrl+PageDown', messages.SHORTCUT_NEXT_TAB],
     ['Ctrl+PageUp', messages.SHORTCUT_PREV_TAB],
-    ['CmdOrCtrl+9', messages.SHORTCUT_SET_ACTIVE_FRAME_TO_LAST]
+    ['CmdOrCtrl+9', messages.SHORTCUT_SET_ACTIVE_FRAME_TO_LAST],
+    ['Alt+Left', messages.SHORTCUT_ACTIVE_FRAME_BACK],
+    ['Alt+Right', messages.SHORTCUT_ACTIVE_FRAME_FORWARD]
```

Comment:
This looks right to me but we need to have these conditionally added w/ another `if (!isDarwin)` because on OS X alt + left goes to previous workd and alt + right goes to next word.